sdk/servicebus/service-bus/test/serviceBusClient.spec.ts

```diff
@@ -943,63 +885,6 @@ describe("Errors after close()", function(): void {
   describe("Errors after close() on senderClient", function(): void {
     const entityToClose = "senderClient";
 
-    it("Partitioned Queue: errors after close() on senderClient", async function(): Promise<void> {
-      await beforeEachTest(
-        TestClientType.PartitionedQueue,
-        TestClientType.PartitionedQueue,
-        entityToClose
-      );
-      await testSender(
-        getSenderClosedErrorMsg(senderClient.entityPath, ClientType.QueueClient, true)
-      );
-      await testCreateSender(getClientClosedErrorMsg(senderClient.entityPath));
-    });
-
-    it("Partitioned Queue with sessions: errors after close() on senderClient", async function(): Promise<
-      void
-    > {
-      await beforeEachTest(
-        TestClientType.PartitionedQueueWithSessions,
-        TestClientType.PartitionedQueueWithSessions,
-        entityToClose,
-        true
-      );
-
-      await testSender(
-        getSenderClosedErrorMsg(senderClient.entityPath, ClientType.QueueClient, true)
-      );
-      await testCreateSender(getClientClosedErrorMsg(senderClient.entityPath));
-    });
-
-    it("Partitioned Topic: errors after close() on senderClient", async function(): Promise<void> {
-      await beforeEachTest(
-        TestClientType.PartitionedTopic,
-        TestClientType.PartitionedSubscription,
-        entityToClose
-      );
-
-      await testSender(
-        getSenderClosedErrorMsg(senderClient.entityPath, ClientType.TopicClient, true)
-      );
-      await testCreateSender(getClientClosedErrorMsg(senderClient.entityPath));
-    });
-
-    it("Partitioned Topic with sessions: errors after close() on senderClient", async function(): Promise<
-      void
-    > {
-      await beforeEachTest(
-        TestClientType.PartitionedTopicWithSessions,
-        TestClientType.PartitionedSubscriptionWithSessions,
-        entityToClose,
-        true
-      );
-
-      await testSender(
-        getSenderClosedErrorMsg(senderClient.entityPath, ClientType.TopicClient, true)
-      );
-      await testCreateSender(getClientClosedErrorMsg(senderClient.entityPath));
-    });
-
```

Comment:
For the `errors after close() on senderClient` case, the sessions variations can be skipped for the same reason discussed elsewhere in the PR that sessions & no-sessions makes no difference when sending as far as client code is considered.

Reply:
Agree, updated

As discussed earlier, let's hold off on this PR until 2 other PRs get some traction - 6603 and 6390